docs/mutating-webhook/configuration.md

```diff
@@ -170,11 +193,13 @@ helm upgrade --install mysql stable/mysql \
 You can also specify a default secret being used by the webhook for cases where a pod has no `imagePullSecrets` specified. To make this work you have to set the environment variables `DEFAULT_IMAGE_PULL_SECRET` and `DEFAULT_IMAGE_PULL_SECRET_NAMESPACE` when deploying the vault-secrets-webhook. Have a look at the values.yaml of the
 [vault-secrets-webhook](https://github.com/banzaicloud/bank-vaults/blob/master/charts/vault-secrets-webhook/values.yaml) helm chart to see how this is done.
 
-**NOTE**: _If you EC2 nodes are having ECR instance role added the webhook can request an ECR access token through that role automatically, instead of an explicit `imagePullSecret`_
+> Note:
+> - If your EC2 nodes have the ECR instance role, the webhook can request an ECR access token through that role automatically, instead of an explicit `imagePullSecret`
+> - If your workload is running on GCP nodes, the webhook automatically authenticates to GCR.
 
 Future improvements:
 
-- on Azure/Alibaba and GKE get a credential dynamically with the specific SDK (for AWS ECR this is already done)
+- on Azure/Alibaba get a credential dynamically with the specific SDK (for AWS ECR this is already done)
```

Comment:
Please GCR as well, it is also implemented by now.

Reply:
```suggestion
- on Azure/Alibaba get a credential dynamically with the specific SDK (for AWS ECR and GCP GCR this is already done)
```